model/src/main/proto/exploration.proto

```diff
@@ -133,7 +131,6 @@ message SubtitledUnicode {
   string content_id = 2;
 }
 
-
 // Structure to support UnicodeOrNone
 message UnicodeOrNone {
```

Comment:
Do we still need this structure? Have all of the ones we actually don't need been removed?

Reply:
No we don't need this anymore.

Reply:
Resolving this on behalf of Ben